Firmware/language.c

```diff
@@ -236,7 +236,9 @@ const char* lang_get_name_by_code(uint16_t code)
 #ifdef COMMUNITY_LANG_GROUP1_LT
 	case LANG_CODE_LT: return _n("Lietuviu"); //community Lithuanian contribution
 #endif // COMMUNITY_LANG_GROUP1_LT
-
+#ifdef COMMUNITY_LANG_RO 
```

Comment:
To be able to support many languages we have to split these at some point in groups and this should be for now `COMMUNITY_LANG_GROUP1_RO`

Reply:
You got it correct here https://github.com/prusa3d/Prusa-Firmware/pull/3343/files#diff-bcba18db116b220c2d9839b91947975b493af75c398d87ae38387546df81db2fR122

Reply:
Still open issue, saw your change https://github.com/prusa3d/Prusa-Firmware/pull/3343/commits/bb4fc2ed50cd22d9340dfe0c90456f20526bfdeb but you forgot about this one.